discord_bots/cogs/economy.py

```diff
@@ -47,6 +47,8 @@
 )
 from discord_bots.utils import short_uuid
 
+_log = logging.getLogger(__name__)
```

Comment:
not a blocker but might be nice if this belonged to the class? then the log would say `EconomyCommands`. we also might be able to to add it to the `BaseCog` so we don't have to type it everywhere

Reply:
Definitely, I think that could be refactored easily in the future. This at least adds some more context in the meantime. One issue I have with BaseCog, is that it uses `cog_before_invoke` and assumes that it's dealing with `Contexts`. It wouldn't work well with `Interactions` without some refactoring, so I didn't want to touch it for now since it's in use for message commands still.